spec/controllers/task_managements_controller_spec.rb

```diff
@@ -33,12 +36,15 @@
       end
       it "assigns tasks created to tasker" do
         get :index
-        expect(assigns(:tasks)).to eql @user.tasks_created.to_a
+
+        assign_tasks
+        expect(@tasker.tasks_created).not_to be_blank
```

Comment:
Nice work so far. It will be great to test for the exact form of the task created instead of a not blank. Something like the count of the tasks assigned.